apps/coscrad-frontend/src/components/footer/footer.tsx

```diff
@@ -22,6 +25,27 @@ export const Footer = (): JSX.Element => {
                 backgroundColor: '#ededed',
             }}
         >
+            <Grid>Phone: {phoneNumber}</Grid>
+            <Grid>Address: {address}</Grid>
+            <Grid item>
+                {internalLinks.map(({ url, iconUrl, description }) => (
+                    // <Tooltip title={description}>
+                    <IconButton
+                        href={url}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                        color="primary"
+                    >
+                        {/* <img width={100} src={iconUrl} alt={description} /> */}
```

Comment:
You'll still need some custom icons. You may find [this](https://mui.com/material-ui/icons/#svgicon) helpful.